backstory/src/test/java/com/google/sps/story/StoryManagerTest.java

```diff
@@ -85,6 +85,19 @@ public void invalidTemperature() {
     storyManager = new StoryManagerImpl(PREFIX_SAMPLE, SIZE_SAMPLE, 1000.0);
   }
 
+  @Test
+  public void cyclePastBounds() {
+    storyManager = new StoryManagerImpl(PREFIX_SAMPLE, SIZE_SAMPLE, TEMPERATURE_SAMPLE);
+    try{
+      for(int index = 0; index < 10; index++) {
```

Comment:
The value of `10` here requires that this test case have internal knowledge of the number of URLs StoryManager may use. This would then require maintenance if the number of URLs changes. What would be a away to avoid that type of maintenance?

Reply:
Changed to return boolean for indication of last URL.